gcp/k8s/secret-volume.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1
+kind: SecretProviderClass
+metadata:
+  name: wrongsecrets-gcp-secretsmanager
+spec:
+  provider: gcp
+  parameters:
+    secrets: |
+      - resourceName: "projects/owasp-wrongsecrets/secrets/wrongsecret-1/versions/latest"
+        fileName: "wrongsecret"
+      - resourceName: "projects/owasp-wrongsecrets/secrets/wrongsecret-2/versions/latest"
+        fileName: "wrongsecret-2"
```

Comment:
We probably want to add this file to gitignore

Reply:
Done